docs/start/setup-channel.mdx

```diff
@@ -165,8 +165,73 @@ print(result)
     from qiskit_ibm_runtime import QiskitRuntimeService
 
     service = QiskitRuntimeService()
+  ```
+
+## Access IBM Quantum Systems using REST API
```

Comment:
As you don't yet have the version of this example that explains how to access the ibm cloud with the REST api I think it makes the most sense to move this section up so that it is in between "Set up to use IBM Quantum Platform" and "Set up to use IBM Cloud". Also I think the title of this section should be something like "Set up to use IBM Quantum Platform with REST API" or similar, to better match the format of the other sections